### ISSUE TITLE
Insert generated content earlier

### ISSUE THREAD

**Issue by RodentOfUnusualSize** [RodentOfUnusualSize/random-message-loader#4]
**Is your feature request related to a problem? Please describe.**
Right now, the script waits until ***everything*** a page needs is loaded: scripts, images, styles, *everything*.

It would be cool if the script could insert messages as soon as possible, rather than waiting for *everything*.

**Describe the solution you'd like**
Right now, the script waits on the `load` event. It really only needs to wait for the `DOMContentLoaded` event.

**Describe alternatives you've considered**
None.

**Additional context**
The tricky part is that we don’t want to *miss* the `DOMContentLoaded` event.

With the `load` event, there is no possible way to avoid missing the event, even when loading the script asynchronously. However, with `DOMContentLoaded`, the script may be loaded after the DOM, which means we’ve missed the event. We need to make sure that doesn’t happen.

**Comment by RodentOfUnusualSize** [comment]
This issue is waiting on #5 to be resolved first.

**Comment by RodentOfUnusualSize** [comment]
Done. Merged into main with commit f66fe5fdea21a45db25a72457d2e9274f6f1f4c1.
